.gitignore

```diff
@@ -15,3 +15,5 @@ yarn-error.log*
 # misc
 .DS_Store
 temp
+
+.config.json
```

Comment:
should we use a `.env` for consistency with the other repos? added bonus being that it's more straightforward (and IMO more expected) to add environment variables to the pipeline?